ui/pages/add-token/token-list/token-list.component.js

```diff
@@ -35,13 +37,17 @@ export default class TokenList extends Component {
           {Array(6)
             .fill(undefined)
             .map((_, i) => {
-              const { logo, symbol, name, address } = results[i] || {};
+              const { iconUrl, symbol, name, address } = results[i] || {};
+              // token from dynamic api list is fetched when useTokenDetection is true
+              const iconPath = useTokenDetection
+                ? iconUrl
+                : `images/contract/${iconUrl}`;
```

Comment:
sorry to keep asking for comments but can you explain why in the static list case we need this string interpolation?

Reply:
In the static list, the iconUrl will be holding only a filename for the image, we have a pre-existing setup that will make the images available in the `images/contract/` location (I think created when the contract-metadata package is added to the repo) so that it can be accessed using the filename in iconUrl.

Reply:
Would you mind adding this as a comment in the code?

Reply:
sure, will add.